pyproject.toml

```diff
@@ -46,3 +46,16 @@ py-modules = ["adafruit_ili9341"]
 [tool.setuptools.dynamic]
 dependencies = {file = ["requirements.txt"]}
 optional-dependencies = {optional = {file = ["optional_requirements.txt"]}}
+
+[tool.ruff]
```

Comment:
We don't generally want to make configuration changes to pyproject.toml in single repos like this. 

We have `ruff.toml` file with relatively configuration inside of it that is used for all of the library repos.

If there are configuration changes that we should consider then we'll want to discuss doing so more broadly with a patch across all repos instead of having it changed one-off in PRs like this.

Please revert the changes in pyproject.toml for now.